support/notule-exporter.js

```diff
@@ -22,7 +22,18 @@ async function extractNotulenContentFromZitting(zitting, publicBehandelingUris)
   let behandelingsHtml = '';
   behandelingsHtml = generateBehandelingHtml(zitting, publicBehandelingUris);
   const notulenData = Object.assign(zitting, {behandelingsHtml, prefixes: prefixes.join(' ')});
-  return generateNotulenHtml(notulenData);
+  const html = generateNotulenHtml(notulenData);
+  const errors = [];
+  if(!zitting.geplandeStart.value) {
```

Comment:
this check is quite similar from the one in wrapZittingInfo, might make sense to abstract it and reuse in both places